app/components/ui/atoms/ActionText.md

```diff
@@ -0,0 +1,8 @@
+A portion of text that has an action associated with clicking it.
+
+```js
+      <ActionText onClick={alert('do action')}>
```

Comment:
`onClick` should be a function.

The alert pops up as soon as you open the styleguide currently.

Reply:
Ooops